content/docs-v1/page-content.md

```diff
@@ -104,37 +108,33 @@ An ID and the # symbol is also used for other kinds of elements on the same page
 
 - For example, the ID of a heading will often be the name of that heading.
 
-    ```md
-    [referencetolink](#element-id)
-    See [heading 1](#heading-1).
-    ```
+```md
+[referencetolink](#element-id)  
+See [heading 1](#heading-1).
+```
 
 #### Links to Elements on a Separate Page
 
 Following the formula for internal links between pages, you can also specify an element on a separate page as a link destination by adding the # symbol and the element’s ID on to the end of a page link.
 
-      ```  md
-      [referencetolink](/nameofpage/#idname)
-      See the introduction [notes](/introduction/#notes)
-      ```
-
-{{< q-class "box warning" >}}
-- Blackfriday, Quire’s built in Markdown processor, will incorrectly create link when there is some text in brackets followed immediately by more text in parentheses even if there is a space between them. To avoid the linking, you can use a \ (backslash) escape character before the first parentheses, such as: `[not a link] \(1926)`  The \ will not display in the final rendered text.
-{{< /q-class >}}
+```md
+[referencetolink](/nameofpage/#idname)
+See the introduction [notes](/introduction/#notes)
+```
 
 #### Linked Footnotes
 
-When creating [footnotes with Markdown](/docs-v1/fundamentals/#footnotes), links are automatically created between the footnote number in the text and the note itself at the bottom of the page. To link to a note from other locations, you can use its automatically generated ID, which always follows the format `fn:#` where # is the number of the footnote.
+When creating [footnotes with Markdown](/docs-v1/fundamentals/#footnotes), links are automatically created between the footnote number in the text and the note itself at the bottom of the page. To link to a note from other locations, you can use its automatically generated ID, which always follows the format `fn#` where # is the number of the footnote.
 
 ```md
-[referencetolink](#fn:#)
-Also in regards to [note 21](#fn:21)
-See [note 3, chapter 2](/chapter-2/#fn:3)
+[referencetolink](#fn#)
+Also in regards to [note 21](#fn21)
+See [note 3, chapter 2](/chapter-2/#fn3)
 ```
 
 #### Citations
 
-When the citation shortcode `{{</* q-cite "author date" "page #" */>}}` is used in a body of text and corresponds to the short and full bibliographic information provided in the `references.yaml` file, an in-page bibliography will be generated and linked to. This linking is completed automatically.
+When the citation shortcode `{% cite "author date" "page #" %}` is used in a body of text and corresponds to the short and full bibliographic information provided in the `references.yaml` file, an in-page bibliography will be generated and linked to. This linking is completed automatically.
```

Comment:
We didn't have this complete in the Hugo version of the docs. The `cite` shortcode accepts three values, not two: `{% cite "author date" "page #" "display text" %}`

Semi-side-note: Matthew et al. generally use single straight quotes in shortcodes and I actually think it looks slightly cleaner: `{% cite 'author date' 'page #' 'display text' %}`. What do you think?

If it is better, we should be consistent both in the docs and in the starter kit whenever we clean it up

Reply:
ah, ok. I was wondering about that. It does look cleaner. I'll just need to go back and fix in various places. 

I will add the info about the three citation shortcode values to the citation section of the docs.